frontends/PyCDE/src/pycde/module.py

```diff
@@ -292,11 +295,11 @@ def outputs() -> list[(str, mlir.ir.Type)]:
   # subclasses. Add the names to "don't touch" since they can't be touched
   # (since they implictly call an OpView property) when the attributes are being
   # scanned in the `mod` constructor.
-  for (idx, (name, _)) in enumerate(mod._input_ports):
+  for (idx, (name, type)) in enumerate(mod._input_ports):
     setattr(
         mod, name,
-        property(lambda self, idx=idx: OpOperand(self, idx, self.operands[idx],
-                                                 self)))
+        property(lambda self, type=type, idx=idx:
```

Comment:
Is `type` being used anywhere? I believe `OpOperandConnect` is just going to take the type from the `Value` passed as the third argument to the constructor.

Reply:
Development cruft. Good catch.